meta-lmp-bsp/recipes-security/optee/optee-os-fio_3.10.0.bbappend

```diff
@@ -15,6 +15,11 @@ EXTRA_OEMAKE_append_apalis-imx6 = " \
     CFG_TZDRAM_START=0x4e000000 CFG_OVERLAY_ADDR=0x16000000 \
     CFG_OVERLAY_RESERVED_MEMORY_ADDRESS_CELLS=1 CFG_OVERLAY_RESERVED_MEMORY_SIZE_CELLS=1 \
 "
+EXTRA_OEMAKE_append_apalis-imx6-sec = " \
+    CFG_REE_FS=n CFG_RPMB_FS=y CFG_RPMB_FS_DEV_ID=2 \
+    CFG_EARLY_TA=y \
+    CFG_IN_TREE_EARLY_TAS=fiovb/22250a54-0bf1-48fe-8002-7b20f1c9c9b1 \
+"
```

Comment:
This seems to go on the previews commit.
I mean, if the purpose is to match with meta-lmp-bsp/recipes-security/optee/optee-os-fio-mfgtool_3.10.0.bbappend commits. Would you, please, double check?

Reply:
The optee-os-fio-mfgtool changes enable writing the RPMB key from mfgtool, while this enables the optee usage from uboot and rootfs, so I'm not sure. I'll wait for others to comment and check if this needs to be changed.